lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -38,7 +38,7 @@ def create_intent(money, payment_method, options = {})
               return result if result.is_a?(ActiveMerchant::Billing::Response)
             end
 
-            add_network_token_cryptogram_and_eci(post, payment_method)
+            add_network_token_cryptogram_and_eci(post, payment_method, options)
```

Comment:
Does Apple Pay and Google Pay also add these values in `add_digital_wallet`? It seems a bit confusing if the new AP route hits this method twice.

Reply:
Good catch! Fixed!